source/linux/Foundational_Components/U-Boot/Apps-Splash-Screen.rst

```diff
@@ -10,247 +10,62 @@ U-Boot Splash Screen
 .. rubric:: AM335x
    :name: am335x
 
-All the code below is based on Processor Linux SDK 03.02.00..05.
+A splash screen is an introductory screen the user sees over the display when the device boots up. Splash screens are typically used to provide feedback to the user that the board is booting up and to showcase
+the vendor logo which helps with branding and identification of which boot image is being used for the bootup.
+All the code related to u-boto splash screen support is based on Processor Linux SDK 09.03.05.02.
 
-There is a frame buffer driver for am335x in the drivers/video directory
-called am3355x-fb.c. It makes calls to routines in board.c to set up the
-LCDC and frame buffer. To use it:
+For AM335x, splash screen is enabled in u-boot which uses the LCDC driver.
+On AM335x evm, the LCD backlight is controlled using enhanced Auxiliary(eCAP0) PWM.
+To use this, enable following configurations in configs/am335x_evm_defconfig.
 
-Either create a new defconfig in the configs directory or just add
-SPLASH to CONFIG\_SYS\_EXTRA\_OPTIONS. In this example the
-am335x\_evm\_defconfig is copied into a new one called
-am335x\_evm\_splash\_defconfig.
+.. code-block:: kconfig
 
-.. code-block:: text
+    CONFIG_SPLASH_SCREEN=y
```

Comment:
The indentation of code sections should clearly line up with code-block directive as mentioned in CONTRIBUTING guidelines - https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace

Reply:
I will fix this.